src/containers/reports/CrisisReportView.js

```diff
@@ -273,7 +277,11 @@ class CrisisReportView extends React.Component<Props, State> {
     history.push(HOME_PATH);
   }
 
-  handleDelete = () => {}
+  handleDelete = () => {
+    const { actions, match } = this.props;
+    const reportEKID :?UUID = match.params[REPORT_ID_PARAM.substr(1)];
```

Comment:
I wonder if it would be cleaner to remove the colon from REPORT_ID_PARAM and include it in the path (or add a second constant that's like REPORT_ID_PATH that's just ``:${REPORT_ID_PARAM}`` because it seems valuable to be able to access the actual param without having to do this kind of manipulation on it

Reply:
This sounds good.